packages/mui-material/src/index.d.ts

```diff
@@ -172,6 +172,9 @@ export * from './Container';
 export { default as CssBaseline } from './CssBaseline';
 export * from './CssBaseline';
 
+/**
+ * @deprecated use color-scheme in CssBaseline or ScopedCssBaseline to modify scrollbar color
```

Comment:
```suggestion
 * @deprecated the dark scrollbar is automatically added by the `CSSBaseline` and `ScopeCSSBaseline` using the color-scheme attribute
```

And seeing this, it is a breaking change. Could we maybe add a prop to enable this? Something like `setColorScheme`. cc @siriwatknp what do you think?

Reply:
> Could we maybe add a prop to enable this

a prop to which component?

Reply:
Ah, I got it. a prop to `CssBaseline`. Yes, that make sense.

```js
<CssBaseline enableColorScheme />
```
@mnajdova @alexfauquette What do you think?

Reply:
So if I resume, `enableColorScheme` is by default set to `false` such that the behavior does not change when updating the minor version. Dev can set it to `true` in order to use `color-scheme` instead of `darkScrollbar`.

Reply:
Yes, you are right.

Reply:
What's the use case for the opt-in prop? Setting the right color sheme by default sounds like a bug fix to me.
If it's about playing it safe because we are unsure, then I would argue for trying out the path that will maximize our learning: be greedy, and revert if necessary.

Reply:
Regarding the deprecation, we only deprecated it with TypeScript, not with the runtime, is it expected?

At some point I was wondering about turning the helper to be a generic high level scrollbar color customization tool. But I'm not sure there is a market for it, it might be overkill 😁.

Reply:
On my side, I have been confused, between the code of the `CssBaseline` and the code of the website.

The idea was that replacing `darkScrollbar` with `color-scheme` could lead to style modification for users. But it is not the case, because `darkScrollbar` is not set by default in the theme. 

So if we set `colorScheme: theme.palette.mode` as the default behavior of the CssBaseline the following should happen:
- For projects using `darkScrollbar` as proposed in the doc, the scollbar will still be override
- For projects not using `darkScrollbar` scrollbar colors will be override (which is an improvement)

Then, it can be seen as a clean bug fix replacing a hack